napari_skimage_regionprops/_load_csv.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-from napari_tools_menu import register_function
+from ._register_function import register_function
```

Comment:
Hi @tcompa ,

wouldn't it make sense to fix the underlying issue in napari-tools-menu instead of replacing this line in many places?

Reply:
That would make sense indeed, but I need to look at napari-tools-menu more carefully.

The rationale for my change was that the napari-tools-menu module is only used in the presence of some QT module, and I thought it was less important to make it work in cases where there is no napari or qtpy available.
On the other hand, I guess it is imported in many of your napari-related modules, and then if we solve this "problem" in napari-tools-menu it is already solved for all the other modules, which is a good idea.

Let's then put this PR on hold, and let's see whether we can get napari-tools-menu to also work without qt/napari. 

---

At a first look, if we want to use the `register_function` decorator (https://github.com/haesleinhuepf/napari-tools-menu/blob/fc11b5434b22c201903e05810a278d01f3b24fc1/napari_tools_menu/__init__.py#L170-L173) then there needs to be a `ToolsMenu` object with a dictionary attribute `menus`. Having this available without importing `qtpy.QtWidgets.QMenu` can be achieved in a way similar to the one I used now for `TableWidget`. That is, in napari-tools-menu, I could add
```python
try:
    from qtpy.QtWidgets import QMenu
except ModuleNotFoundError:
    print("some warning")
    class Qmenu:
        menus = {}
```
so that the `register_function` can still be imported and used in napari-skimage-regionprops. I'll open a PR quickly, just for the sake of discussion (probably next week).